package.json

```diff
@@ -7,20 +7,22 @@
     "mysql": "^2.11.1"
   },
   "devDependencies": {
-    "babel": "^6.1.18",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.24.1",
```

Comment:
its not but its needed to run an npm script that uses the `babel` command. I'll just remove the command since we are instructing to use a global babel-cli